sass/main.css

```diff
@@ -1 +1,118 @@
-/*# sourceMappingURL=main.css.map */
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  background-color: #111111;
```

Comment:
use $primary-color variable here

Reply:
actually these are .css file and i am working on .scss file,
.scss files are managed automatically by vs code extension.